common/main/es_GT.xml

```diff
@@ -269,7 +269,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				</currencyFormat>
 			</currencyFormatLength>
 			<unitPattern count="one" draft="contributed">↑↑↑</unitPattern>
-			<unitPattern count="other" draft="contributed">{1} {0}</unitPattern>
```

Comment:
Shouldn't we keep the other form here since it's what's being used currently?

Reply:
Grammatically I figured there was no reason for es-GT to diverge from es-419 or es, all of which inherit from root; do we even have vetters for es-GT?

Reply:
Not recently, I'm not sure when/why that change was made. I'm assuming it's very old.